assets/scss/_variables_project.scss

```diff
@@ -488,20 +560,31 @@ ol.breadcrumb li:not(:first-child):before {
 
 // Table of Contents Component
 #TableOfContents {
-    
+    font-family: $base-font;
     font-size: 13px !important;
-    font-family: "Source Sans Pro", sans-serif;
     a{
         margin: 0px !important;
         padding: 8px !important;
         font-weight: 500 !important;
+        font-size: 13px !important;
     }
     // List styles
     ul { 
         padding: 0px !important;
         li {
             padding: 0px !important; 
             line-height: 14px;
+            font-size: 13px !important;
```

Comment:
Same feedback here about the font size. Probably all of the links in the navigation are in `<ul><li>` hierarchy.

Reply:
Right, removed this definition; everything should be styled by just the font-size definition in `a` now